includes/RestApi/Themes/ThemeVariationsController.php

```diff
@@ -67,4 +84,25 @@ public function get_theme_variations( \WP_REST_Request $request ) {
 		);
 	}
 
+	/**
+	 * Saves the custom active theme variations.
+	 *
+	 * @return \WP_REST_Response|\WP_Error
+	 */
+	public function set_theme_variation(\WP_REST_Request $request)
+	{
+		// The theme data with the new Colors and Fonts
+		$theme_data = json_decode($request->get_body(), true);
```

Comment:
Should we validate this similar to flow data to ensure we do not introduce anything unnecessary?

Reply:
Cool let me do that.

Reply:
The validation can move inside a `validate_callback` function as part of the endpoint registration https://www.shawnhooper.ca/2017/02/15/wp-rest-secrets-found-reading-core-code/

Reply:
Done